features/supplier/brief_response.feature

```diff
@@ -2,9 +2,10 @@
 Feature: Supplier applies for a brief
 
 Background:
+  Given I have a live digital outcomes and specialists framework
   Given I have a buyer
   Given I have a supplier
```

Comment:
Yes,  given/when/then should appear once and following lines should begin with and